src/styles/catalog.scss

```diff
@@ -0,0 +1,26 @@
+.catalog {
+  padding: 50px 40px;
+  display: grid;
+  grid-template-columns: repeat(1, 200px);
```

Comment:
It seems you have hard-coded the width of the grid items to be 200px. This can cause issues with responsiveness, especially on smaller screens. Consider using a relative unit like 'fr' or 'auto' for better responsiveness.